src/components/ProjectCard/index.jsx

```diff
@@ -0,0 +1,103 @@
+import {
+  withStyles,
+  Card,
+  CardContent,
+  CardActions,
+  Typography,
+} from 'material-ui';
+import { Component } from 'react';
+import { object, node, string } from 'prop-types';
+import ReactMarkdown from 'react-markdown';
+
+@withStyles(() => ({
+  card: {
+    marginTop: '30px',
+    textAlign: 'center',
+    display: 'inline-block',
+    position: 'relative',
+    width: '100%',
+    margin: '25px 0',
+    boxShadow: '0 1px 4px 0 rgba(0, 0, 0, 0.14)',
+    borderRadius: '3px',
+    color: 'rgba(0, 0, 0, 0.87)',
+    background: '#fff',
+    minHeight: '250px',
+  },
+  textAlign: {
+    textAlign: 'center',
+  },
+  cardSubtitle: {
+    color: '#999999',
+    fontFamily: '"Roboto", "Helvetica", "Arial", sans-serif',
+    fontWeight: '300',
+    lineHeight: '1.5em',
+    fontSize: '1em',
+    textTransform: 'uppercase',
+    marginTop: '10px',
+    marginBottom: '10px',
+  },
+  cardTitle: {
+    fontFamily: '"Roboto", "Helvetica", "Arial", sans-serif',
+    fontWeight: '300',
+    lineHeight: '1.5em',
+    fontSize: '1.3em',
+    marginTop: '10px',
+    marginBottom: '10px',
+  },
+  cardDescription: {
+    fontFamily: '"Roboto", "Helvetica", "Arial", sans-serif',
```

Comment:
Remove this line. Adding a variant to the `Typography` component will apply this style.